docs/about.md

```diff
@@ -5,21 +5,54 @@ hide:
 
 # Data Science in NHS England
 
-We are the NHS England Data Science Team with a vision: 
+<div markdown>
+
+![Image title](../images/DS_team_photo.jpeg){ width="450" alt-tex="Picture of the Data Science team stood on some steps in London."  align=right }
```

Comment:
do you reckon youd be able to compress the image so it loads better? Otherwise I'll do it at some point

Reply:
Done